rtslib/fabric.py

```diff
@@ -240,7 +240,11 @@ def _set_discovery_enable_auth(self, enable):
         self._check_self()
         self._assert_feature('discovery_auth')
         path = "%s/discovery_auth/enforce_discovery_auth" % self.path
-        if int(enable):
+        if enable == 'True':
+            enable = 1
+        elif enable == 'False':
+            enable = 0
+        elif int(enable):
```

Comment:
I do not think the user know about python booleans always, so if they do false/true instead of False/True we will crash here due to int() not being able to convert it. I think it needs to be a enable.isdigit() or maybe a try/except ValueError to detect bad input. Whatever we normally do for input validation I guess.